package/MDAnalysis/analysis/align.py

```diff
@@ -196,10 +196,14 @@
 
 import numpy as np
 
-import Bio.SeqIO
-import Bio.AlignIO
-import Bio.Align
-import Bio.Align.Applications
+try:
+    import Bio.AlignIO
+    import Bio.Align
+    import Bio.Align.Applications
```

Comment:
Mmmh, why is this not covered?

Reply:
Not sure - I added a quick test in [5529a5d](https://github.com/MDAnalysis/mdanalysis/pull/4332/commits/5529a5d7ce7f627f9e58af80d3374d735a637638) to demonstrate that the test was being called, and it is. This, like the networkx issue, seems to be a codecov thing?

Reply:
Ok, thanks for checking. I thought it might be the same issue you outlined with NetworkX

Reply:
Can confirm at least two hits on codecov: https://app.codecov.io/gh/MDAnalysis/mdanalysis/pull/4332?src=pr&el=h1&utm_medium=referral&utm_source=github&utm_content=checks&utm_campaign=pr+comments&utm_term=MDAnalysis